packages/chat-core-zendesk/src/infra/ChatCoreZendeskImpl.ts

```diff
@@ -47,12 +47,14 @@ export class ChatCoreZendeskImpl {
   private eventListeners: { [T in keyof EventMap]?: EventCallback<T>[] } = {};
   private conversationId: string | undefined;
   private integrationId: string;
+  private tags: string[] = ["yext-chat-agent-handoff"];
 
   constructor(config: ChatCoreZendeskConfig) {
     if (window === undefined) {
       throw new Error("This package can only be used in the browser.");
     }
     this.integrationId = config.integrationId;
+    this.tags = [...this.tags, ...(config.ticketTags ?? [])];
```

Comment:
does this break if the config somehow end up providing the same tag as the private tags?

Reply:
Doesn't appear to break anything, but on the client-side there are duplicate tags. Zendesk seems to handle this fine (deduped on their end) but just to be safe I added deduping logic below this line.